src/armTemplates/resources/functionApp.test.ts

```diff
@@ -80,7 +80,7 @@ describe("Function App Resource", () => {
         const config = getConfig(FunctionAppOS.LINUX, Runtime.PYTHON36);
 
         const resource = new FunctionAppResource();
-  
```

Comment:
Let's not introduce whitespace and unrelated changes

Reply:
unrelated changes like the tests `it("throw exception on get runtime version")` and `it("throw exception on get runtime language")`

Reply:
@lcsales98 see line under which I've put comment. There should be no change here